Makefile

```
@@ -246,22 +205,7 @@ build-dirs:
 run: build-dirs
 ifeq ($(DOCKER_BUILD),"true")
 	@echo "running CMD in the containerized build environment"
-	@docker run                                                     \
-		--platform $(PLATFORM)                                      \
-		--rm                                                        \
-		--net host                                                  \
-		-e GITHUB_TOKEN=$(GITHUB_TOKEN)                             \
-		-v "${HOME}/.kube:/root/.kube"                              \
-		-v "$(PWD)/.go/pkg:/go/pkg"                                 \
-		-v "$(PWD)/.go/cache:/go/.cache"                            \
-		-v "$(PWD):/go/src/$(PKG)"                                  \
-		-v "$(PWD)/bin/$(ARCH):/go/bin"                             \
-		-v "$(PWD)/.go/std/$(ARCH):/usr/local/go/pkg/linux_$(ARCH)" \
-		-v "$(DOCKER_CONFIG):/root/.docker"                         \
-		-v /var/run/docker.sock:/var/run/docker.sock                \
-		-w /go/src/$(PKG)                                           \
-		$(BUILD_IMAGE)                                              \
-		/bin/bash $(CMD)
+	@PWD=$(PWD) ARCH=$(ARCH) PKG=$(PKG) GITHUB_TOKEN=$(GITHUB_TOKEN) CMD="$(CMD)" /bin/bash ./build/run_container.sh build
```

```suggestion
	@PWD=$(PWD) ARCH=$(ARCH) PKG=$(PKG) GITHUB_TOKEN=$(GITHUB_TOKEN) CMD="$(CMD)" /bin/bash ./build/run_container.sh build
```

what do you think about renaming `build` to `run`?

---

I suggested this because we are eventually running command using this target and necessarily building always.

---

fixed